src/OneButton.cpp

```diff
@@ -95,6 +95,11 @@ void OneButton::attachClick(parameterizedCallbackFunction newFunction, void *par
   _clickFuncParam = parameter;
 } // attachClick
 
+void OneButton::attachClick(taggetCallbackFunction newFunction)
```

Comment:
Should these instances be `tagged`, not `tagget`?

```suggestion
void OneButton::attachClick(taggedCallbackFunction newFunction)
```